freetextresponse/freetextresponse.py

```diff
@@ -223,6 +236,11 @@ def validate_field_data(self, validation, data):
                 'Minimum Word Count cannot be greater than Max Word Count'
             )
             validation.add(msg)
+        if not data.submitted_message:
+            msg = FreeTextResponse._generate_validation_message(
+                'Submission Received Message cannot be blank'
```

Comment:
looking at the source code, it does not:
https://github.com/edx/XBlock/blob/master/xblock/validation.py
I will i18n the messages in _generate_validation_message